File: Selenium/Project/PrjActivity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager

# Set up Firefox driver
GeckoDriverManager().setup()
# Create a new instance of the Firefox driver
driver = webdriver.Firefox()

# Open the page
driver.get("https://alchemy.hguy.co/jobs/")
imageElement = driver.find_element(By.CSS_SELECTOR, "img")
imageUrl = imageElement.get_attribute("src")
print("Header Image URL:", imageUrl)
driver.quit()

File: Selenium/Project/PrjActivity2.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;

import java.net.URL;

public class PrjActivity2 {
    public static void main (String[] args) throws InterruptedException {
        //download the gecko driver, but geckodriver is not downloaded, so download that and provide the path
        WebDriverManager.firefoxdriver().setup();
        System.setProperty("Webdriver.gecko.driver", "C:\\Gecko Driver\\geckodriver.exe");
        //Initialize the firefox driver
        WebDriver driver = new FirefoxDriver();
        //open a page
        driver.get("https://alchemy.hguy.co/crm/");
        //Get the image url
     String src = driver.findElement(By.tagName("img")).getAttribute("src");
     //print the image url
       System.out.println("The image url is " +src);
       driver.quit();
    }
}
